src/routes/object/getPublicObject.ts

```diff
@@ -42,7 +42,7 @@ export default async function routes(fastify: FastifyInstance) {
       schema: {
         params: getPublicObjectParamsSchema,
         summary,
-        response: { '4xx': { $ref: 'errorSchema#' } },
+        response: { '4xx': { $ref: 'errorSchema#', description: 'Error response' } },
```

Comment:
what does this feed into? Is adding a generic 'error response' adding value to a 4xx code?

Reply:
[This page](https://supabase.com/docs/reference/storage/usage) looks better.